mixer/pkg/runtime/dispatcher/dispatcher_test.go

```diff
@@ -96,6 +99,38 @@ ident                         : dest.istio-system
 `,
 	},
 
+	{
+		name: "BasicCheckOutput",
+		config: []string{
+			data.HandlerACheckOutput1,
+			data.InstanceCheckOutput1,
+			data.RuleCheckOutput1,
+		},
+		variety:             tpb.TEMPLATE_VARIETY_CHECK_WITH_OUTPUT,
+		expectedCheckResult: adapter.CheckResult{ValidDuration: 123 * time.Second, ValidUseCount: 123},
+		expectedCheckDirective: &v1.RouteDirective{
+			RequestHeaderOperations: []v1.HeaderOperation{{
+				Name:  "key1",
+				Value: "1337",
+			}},
+			ResponseHeaderOperations: []v1.HeaderOperation{{
+				Name:      "key2",
+				Value:     "1337",
+				Operation: v1.APPEND,
+			}},
+		},
+		log: `
+[tcheckoutput] InstanceBuilderFn() => name: 'tcheckoutput', bag: '---
+ident                         : dest.istio-system
+'
+[tcheckoutput] InstanceBuilderFn() <= (SUCCESS)
+[tcheckoutput] DispatchCheckOutput => context exists: 'true'
+[tcheckoutput] DispatchCheckOutput => handler exists: 'true'
+[tcheckoutput] DispatchCheckOutput => instance:       '&Struct{Fields:map[string]*Value{},XXX_unrecognized:[],}'
+[tcheckoutput] DispatchCheckOutput <= (SUCCESS)
+		`,
+	},
+
```

Comment:
side note: this is not the PR to fix this, but I am growing more and more concerned by the hardcoding of exact text matches in the test cases. Any slight change to even dependencies here will require us to touch *all* of the test data.

Reply:
Yes, but there's too much of it to fix here. Something to be cleaned up later.